src/SFA.DAS.AANHub.Api/Common/ActionResponseControllerBase.cs

```diff
@@ -50,4 +57,12 @@ private static List<ValidationError> FormatErrors(IEnumerable<ValidationFailure>
             ErrorMessage = err.ErrorMessage
         }).ToList();
     }
+
+    private static IReadOnlyCollection<string> Get404WorthyErrorMessages()
+    {
+        return new[]
+        {
+            CreateAttendanceCommandValidator.EventNotFoundMessage,
+        };
```

Comment:
Cannot put specific behaviour in base class. This behaviour is incorrect anyway, you cannot return not found for POST request. It should be BadRequest only. If the CalendarEventId is invalid then it should be a BadRequestResponse

Reply:
Renived